changelog/0077-sql-repeatrecord-pr1.yml

```
@@ -0,0 +1,103 @@
+title: 'SQL Repat Record Migration'
+key: sql-repeatrecord-pr1
+date: 2024-02-13
+optional_per_env: yes
+# (optional) Min version of HQ that MUST be deployed before this change can be rolled out (commit hash)
+min_commcare_version: TODO
+# (optional) Max version of HQ that can be deployed before this change MUST be rolled out (commit hash)
+max_commcare_version:
+context: |
+  Prepare for and migrate Repeat Records from Couch to SQL.
+
+details: |
+  This is an optional procedure that should be done if you use data forwarding
+  and want to ensure a future deploy is not blocked on needing to perform manual
+  migration steps.
+
+update_steps: |
+  Run the following command to determine how many repeat records you have to
+  migrate (in all commands below, replace `<env>` with your environment name):
+
+  ```sh
+  cchq <env> django-manage shell -c '
+  from corehq.motech.repeaters.management.commands.populate_repeatrecords import count_docs;
+  print("Records to migrate:", count_docs())'
+  ```
+
+  The output should include something like the following:
+
+  ```
+  Records to migrate: 125011
+  ```
+
+  A future deploy will be blocked, requiring manual migration steps, if the
+  number of records to migrate is more than 1000. These steps can be skipped
```

nit: Make the reader aware of whether this applies to them by saying how many records the threshold is first.
```suggestion
  If the number of records to migrate is more than 1000, a future deploy will be blocked, requiring manual migration steps. These steps can be skipped
```